Core/MIPS/IR/IRCompVFPU.cpp

```diff
@@ -1179,10 +1227,12 @@ namespace MIPSComp {
 				}
 			}
 			return;
-		} else if (msz == M_4x4 && !IsConsecutive4(sregs)) {
+		} else if (msz == M_4x4 && IsConsecutive4(sregs)) {
+			// Consecutive, which is harder.
```

Comment:
I'll note: NEON also has transpose instructions, and they don't seem terrible.  So even if they were loaded, it's probably doable.  Yeah, that makes sense.

-[Unknown]